packages/react-select/src/components/Option.js

```diff
@@ -80,10 +80,15 @@ const Option = (props: OptionProps) => {
     isFocused,
     isSelected,
     innerRef,
+    type,
     innerProps,
   } = props;
+
   return (
-    <div
+    <li
+      role={type}
+      aria-disabled = {isDisabled}
+      aria-selected={isFocused}
```

Comment:
`isFocused` does not equal a selected option. This should be `isSelected`.

Furthermore, if the option is disabled, this attribute should somehow be omitted:

> If an option is not selectable, omit the aria-selected.
> _[MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role#associated_aria_roles_states_and_properties)_

Reply:
@Rall3n regarding the `aria-disabled` attribute, I believe we want this because the options are not "truly" disabled and we allow the user to click on disabled options so that the information is relayed back to the aria-live region. Let me know if I am missing something.